storage/client/queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -1,29 +1,527 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 package com.azure.storage.queue;
 
-import com.azure.core.ServiceClient;
 import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.http.rest.VoidResponse;
+import com.azure.core.util.Context;
+import com.azure.storage.common.credentials.SASTokenCredential;
+import com.azure.storage.common.credentials.SharedKeyCredential;
 import com.azure.storage.queue.implementation.AzureQueueStorageBuilder;
 import com.azure.storage.queue.implementation.AzureQueueStorageImpl;
-
+import com.azure.storage.queue.models.DequeuedMessage;
+import com.azure.storage.queue.models.EnqueuedMessage;
+import com.azure.storage.queue.models.MessageIdUpdateHeaders;
+import com.azure.storage.queue.models.MessageIdsUpdateResponse;
+import com.azure.storage.queue.models.PeekedMessage;
+import com.azure.storage.queue.models.QueueGetPropertiesHeaders;
+import com.azure.storage.queue.models.QueueMessage;
+import com.azure.storage.queue.models.QueueProperties;
+import com.azure.storage.queue.models.QueuesGetPropertiesResponse;
+import com.azure.storage.queue.models.SignedIdentifier;
+import com.azure.storage.queue.models.StorageErrorException;
+import com.azure.storage.queue.models.UpdatedMessage;
+import java.net.MalformedURLException;
 import java.net.URL;
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * This class provides a client that contains all the operations for interacting with a queue in Azure Storage Queue.
+ * Operations allowed by the client are creating and deleting the queue, retrieving and updating metadata and access
+ * policies of the queue, and enqueuing, dequeuing, peeking, updating, and deleting messages.
+ *
+ * <p><strong>Instantiating an Asynchronous Queue Client</strong></p>
+ *
+ * <pre>
```

Comment:
Are we moving all code samples to snippets in this PR or will it be pushed off to another?

(My preference would move them in a follow-up PR)

Reply:
Yes. This is not required for passing the CI. I's suggest to push the current code to feature branch first. We have an issue tracking this in the Epic.

Reply:
Get them moved out into Java files ASAP after merging. This code sample is incorrect and wouldn't compile today, so the sooner it is in Java the sooner it will become current again.